stable/datacube-dashboard/templates/deployment.yaml

```diff
@@ -62,7 +62,6 @@ spec:
         resources:
 {{ toYaml .Values.resources | indent 12 }}
         args: [ {{- range .Values.dockerArgs }} {{ . | quote }}, {{ end -}} ]
-        terminationGracePeriodSeconds: 30
```

Comment:
This should be set as if the service cannot find use cases the job should be terminated

Reply:
Added `terminationGracePeriodSeconds`